### PR TITLE
[harfbuzz] Setup a new target, hb-draw-fuzzer.

### PR DESCRIPTION
**Pull request by ebraminio**
https://github.com/harfbuzz/harfbuzz/pull/2192 for more info

### CODE REVIEW

File: projects/harfbuzz/hb-draw-fuzzer.options [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[libfuzzer]
```
Comment (jonathanmetzman) [inline projects/harfbuzz/hb-draw-fuzzer.options]: You want any time this fuzzer takes longer than 5 seconds to process an input to be filed as a bug?

Comment (ebraminio) [reply projects/harfbuzz/hb-draw-fuzzer.options]: It is what our other fuzzers have also but sure we don't want infinite loop with the API, actually, I find one infinite loop which was a regression while developing this fuzzer and fixed it.

Comment (jonathanmetzman) [reply projects/harfbuzz/hb-draw-fuzzer.options]: ACK

Comment (ebraminio) [reply projects/harfbuzz/hb-draw-fuzzer.options]: Thank you very much :)

**Pull request merged**